packages/api-db-mongodb/src/migration.ts

```diff
@@ -107,7 +107,7 @@ export const Migrations: Migration[] = [
 
       await userRoles.createIndex({name: 1}, {unique: true})
 
-      userRoles.insertMany([
+      await userRoles.insertMany([
```

Comment:
Just that I understand it correctly: setting an await here prevents the memory leakage from occuring?

Reply:
here you are inside a `async` context `async dbMigrate` here, await prefix should be added basically to all functions to ensure, function execution is concatinated correctly.

in general i dont understand why sometime its 
`const migrations = db.collection(CollectionName.Migrations)` or `const migrations = await db.collection(CollectionName.Migrations)`

Just

Reply:
@0815pascal Not really! Because the method `insertMany` is asynchronous the `await` is needed. The memory leakage happened because the connection to the database during a migration never gets closed.